catroid/src/org/catrobat/catroid/content/GroupItemSprite.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2016 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.content;
+
+public class GroupItemSprite extends Sprite {
```

Comment:
I don't know what's best practice here
a) extend a class and check with ``instanceof`` operator if an object is an instance of a certain subclass
b) add an ``enum`` spriteTypes in ``Sprite`` and check ``if (sprite.getSpriteType() == SpriteType.GROUP_ITEM)

I guess the answer to which method is better depends on the likelihood that a subclass could override or add functionality in the future. I don't expect you to change anything here, since it might involve changing lots of "instanceof" checks to type-checks.